library/markdown.php

```diff
@@ -15,7 +15,9 @@ function Markdown($text) {
 	$stamp1 = microtime(true);
 
 	# Read file and pass content through the Markdown parser
```

Comment:
All right, I fixed the formatting, but I won't move this file until we have a generic TextProcessor parent class I can extends to create a Markdown class that will retain the timestamp.